src/main/msp/msp_serial.h

```diff
@@ -39,23 +38,24 @@ typedef enum {
 } mspEvaluateNonMspData_e;
 
 #define MSP_PORT_INBUF_SIZE 64
```

Comment:
I still don't see why we would need this for communication with the configurator. Have you given thought to how to implement this in a simple way in JavaScript?